packages/opentelemetry-test-utils/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@opentelemetry/contrib-test-utils",
-  "version": "0.25.0",
+  "version": "0.26.0",
```

Comment:
Is it intended to move a single package to a new version?

Reply:
Yes. The automation is intended to allow individual package owners to make and release changes without waiting for other packages.

Reply:
ok.
As a result it's most likely better to remove contrib from the [compatibility matrix](https://github.com/open-telemetry/opentelemetry-js#compatibility-matrix) as there is no contrib version anymore.

Reply:
Yes I agree. We are going to have to put in each contrib package's README which SDK versions it is expected to work with.